ci-operator/config/opendatahub-io/notebooks/opendatahub-io-notebooks-main.yaml

```diff
@@ -15,6 +15,32 @@ resources:
     requests:
       cpu: 200m
       memory: 400Mi
+tests:
+- as: base-ubi8-python-3.8-mirror
```

Comment:
Can you update the tests with these names?

- notebook-base-ubi8-python-3.8-image-mirror
- notebook-base-ubi8-python-3.8-pr-image-mirror